vendor/ethcompiler/README.md

```diff
@@ -0,0 +1,27 @@
+# Eth Compiler
+
+Contains a simple image to compile smart contracts easily. Usually used for
+building blockchain challenges for deployment.
+
+## Example usage
+
+```dockerfile
+FROM ghcr.io/downunderctf/docker-vendor/ethcompiler:foundry as build
+
+COPY contracts/ src/
+COPY foundry.toml foundry.toml
+
+RUN /root/.foundry/bin/forge build
+
+FROM ghcr.io/downunderctf/eth-challenge-base-priv/eth-base:latest as dist
```

Comment:
ig it's an example, not sure if you'd want to keep it here

Reply:
Done